scripts/src/main/resources/scripts/command/gcloud

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+GCLOUD_HOME="${DEVON_IDE_HOME}/software/gcloud"
+TOOL_VERSION_COMMAND="${GCLOUD_HOME}/bin/gcloud --version"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
+
+function doConfig() {
+  local gcloud_config_files="${DEVON_IDE_HOME}/conf/.gcloud"
+  local gcloud_config_export="export CLOUDSDK_CONFIG=${gcloud_config_files}"
+  if ! grep -q "${gcloud_config_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+  then
+    doRunCommand "${gcloud_config_export}"
+    echo -e "\n${gcloud_config_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+    doEcho "Location of GClouds's configuration files is set to ${gcloud_config_files}"
+  fi
+}
+
+# the path of gcloud modules must be added to python's sys.path via .pth file
+function doSetPth() {
+  local wsl_path="${GCLOUD_HOME}/lib"
```

Comment:
just a question: what does this path have to do with WSL (Windows Subsystem for Linux)?

Reply:
Should not block PR. Simply renamed it. Done.